docs/sandbox/apps/apprise.md

```diff
@@ -16,13 +16,25 @@ sb install sandbox-apprise
 
 ```
 
-### 2. Setup
+### 2. URL
 
-As configured, the instance runs on the Docker network accessible to other saltbox network containers at <br />
-`http://apprise:8000` <br />
-as well as via the reverse proxy at <br />
-`https://apprise.domain.tld`.
+- To access apprise, visit `https://apprise._yourdomain.com_`
 
-The configured username/password are taken from your Saltbox [`accounts.yml`](../../saltbox/install/install.md#configuration) file located in `/srv/git/saltbox/accounts.yml`
+### 3. Setup
+
+The instance runs on the Docker network accessible to other saltbox network containers at `http://apprise:8000`
+
+The configured username/password are taken from your Saltbox [`accounts.yml`](../../saltbox/install/install.md#__tabbed_2_1) file located in `/srv/git/saltbox/accounts.yml`
+
+A typical apprise URL would look like this:
+
+``` shell
+
+https://apprise._yourdomain.com_/notify?service=discord&title=Hello&body=World
+
+```
+
+!!! note
```

Comment:
Can we move the note to the top and also mention it provides the API in addition to the webui?

Reply:
Sure thing